daemon/api_asserts.go

```diff
@@ -76,9 +76,22 @@ func assertsFindMany(c *Command, r *http.Request, user *auth.UserState) Response
 	if assertType == nil {
 		return BadRequest("invalid assert type: %q", assertTypeName)
 	}
+	jsonResult := false
+	headersOnly := false
 	headers := map[string]string{}
 	q := r.URL.Query()
 	for k := range q {
+		if k == "json" {
+			switch q.Get(k) {
+			case "true":
+			case "headers":
+				headersOnly = true
+			default:
+				return BadRequest(`"json" query parameter when used must be set to "true" or "headers"`)
```

Comment:
in general in this kind of situation we'd support a `"false"` case that would nop